test/proxy/ERC1967/ERC1967Utils.test.js

```diff
@@ -0,0 +1,186 @@
+const { expectEvent, constants } = require('@openzeppelin/test-helpers');
+const { expectRevertCustomError } = require('../../helpers/customError');
+const {
+  getAddressInSlot,
+  setSlot,
+  getSlot,
+  ImplementationSlot,
+  AdminSlot,
+  BeaconSlot,
+} = require('../../helpers/erc1967');
+
+const { ZERO_ADDRESS } = constants;
+
+const ERC1967Utils = artifacts.require('$ERC1967Utils');
+
+const V1 = artifacts.require('DummyImplementation');
+const V2 = artifacts.require('DummyImplementationV2');
+const UpgradeableBeacon = artifacts.require('UpgradeableBeacon');
```

Comment:
This is causing the upgradeable tests to fail because we exclude most proxy contracts: 

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/84db204a4155b6a058a6e51ea2e053c4d29d29c8/scripts/upgradeable/transpile.sh#L29

If we want to do this we would need a new mock contract. Makes sense though given that what we want is to set it to zero and this contract doesn't allow that.

Reply:
Added, thanks!